teams/codebender.md

```diff
@@ -9,15 +9,51 @@ permalink: /teams/codebender/
  - Moon Seokmin ([@bsmmoon](http://moonmoon3281.blogspot.sg/))
-   - Technology Expertise:
+   - Technology Expertise: Web Crawling
  - Alex Yao ([@alexyaoyang](http://alexyy.com/))
    - Technology Expertise: Web Performance
  - Ma Tanghao ([@harryggg](https://blog.nus.edu.sg/matanghao/))
-   - Technology Expertise: AngularJS
+   - Technology Expertise: Ruby on Rails
  - Jiang Sheng ([@gisonrg](http://blog.gisonrg.me/))
    - Technology Expertise: Node.js
 
-## Workplan
+## Workplan for Neuralyzer
+
+* [x] Learn about possible project technologies (not received project details yet) - Week 1
+* [x] Understand project requirements - Week 2
+* [x] Product Presentation in class with high level understanding - Week 3
+* [x] Plan [database details](https://github.com/codebendercc/Neuralyzer/wiki/ORM-Notes) - Week 3
+* [x] Plan [front-end pages details](https://github.com/codebendercc/Neuralyzer/wiki/Pages-the-service-should-provide) - Week 3
+* [x] Learn more about project background from CEO during Code Sprint - Week 3
+* [x] Plan Project specification - Week 3
+* [x] Decide on the [Symfony Bundles to use](https://github.com/codebendercc/Neuralyzer/wiki/Symfony-Bundles) - Week 3
+* [x] Plan [Analytic and Logging](https://github.com/codebendercc/Neuralyzer/wiki/Types-of-Analytics) for project - Week 3
+* [x] Submit first pull request on 1st day of Code Sprint - Week 3
+* [x] Have first pull request merged [Pull Request #1](https://github.com/codebendercc/Neuralyzer/pull/1) (Merged) - Week 3
+* [x] Create one test and merge into main project [First PHPUnit test](https://github.com/codebendercc/Neuralyzer/pull/12) (Merged) - Week 3
+* [x] Setup [Sensiolabs Insight](https://insight.sensiolabs.com/projects/a371effa-c11d-473f-844d-d5f421df79be) for automated push test - Week 3
+* [x] Get started with basic prototype - Week 4
+* [ ] Fix bug related to flashing in iFrame - Week 5
+* [ ] Create test case for successful flashing - Week 5
+* [ ] Create landing page - Week 6
+* [ ] Create test case for landing page - Week 6
+* [ ] Create database for user and tier - Week 6
+* [ ] Create test case for user & tier database - Week 7
+* [ ] Create registration and login page - Week 7
+* [ ] Create test case for login and registration and database linkage - Week 7
+* [ ] Create database for firmware - Week 8
+* [ ] Create test case to ensure that user and firmware database is linked correctly  - Week 8
+* [ ] Create show info page - Week 8
+* [ ] Create test case to ensure show info page shows correct info - Week 9
+* [ ] Create dashboard page - Week 9
+* [ ] Create test case to ensure dashboard page shows correct and links correctly - Week 10
+* [ ] Create log/analytics database - Week 10
+* [ ] Create Analytics page - Week 11
+* [ ] Create test case to ensure Analytic page shows correct analytics - Week 11
+* [ ] Create admin page - Week 12
+* [ ] Create test case for admin page and features - Week 12
+
+## Workplan for Eratosthenis
```

Comment:
Thanks for spotting that typo, fixed in 049e90084a263c8cc41c27ead446af7269a318ab.